.github/workflows/daily.yml

```diff
@@ -4,78 +4,106 @@ on:
+  #    - uses: ./.github/actions/build-setup
+  #    - name: Configure AWS Credentials
+  #      uses: aws-actions/configure-aws-credentials@v1
+  #      with:
+  #        aws-access-key-id: ${{ secrets.DIEM_S3_AWS_ACCESS_KEY_ID }}
+  #        aws-secret-access-key: ${{ secrets.DIEM_S3_AWS_SECRET_ACCESS_KEY }}
+  #        aws-region: ${{ secrets.DIEM_S3_AWS_REGION }}
+  #    - name: produce coverage
+  #      run: cargo xtest --html-cov-dir=$CODECOV_OUTPUT/grcovhtml/ --html-lcov-dir=$CODECOV_OUTPUT/lcovhtml/ --no-fail-fast -j 16 || true
+  #    - name: Push Coverage Reports to S3
+  #      run: |
+  #        set -x
+  #        SUFFIX="$(date +"%Y-%m-%d")-$(git rev-parse --short=8 HEAD)"
+  #        PREFIX="ci-artifacts.diem.com/coverage";
+  #        #Push grcov
+  #        aws s3 cp --recursive ${CODECOV_OUTPUT}/grcovhtml "s3://${PREFIX}/unit-coverage/${SUFFIX}/";
+  #        aws s3 cp --recursive ${CODECOV_OUTPUT}/grcovhtml "s3://${PREFIX}/unit-coverage/latest/";
+  #        echo "Grcov available in s3 https://${PREFIX}/unit-coverage/${SUFFIX}/index.html" >> ${MESSAGE_PAYLOAD_FILE}
+  #        #Push lcov
+  #        aws s3 cp --recursive ${CODECOV_OUTPUT}/lcovhtml "s3://${PREFIX}/lcov-unit-coverage/${SUFFIX}/";
+  #        aws s3 cp --recursive ${CODECOV_OUTPUT}/lcovhtml "s3://${PREFIX}/lcov-unit-coverage/latest/";
+  #        echo "lcov available in s3 https://${PREFIX}/lcov-unit-coverage/${SUFFIX}/index.html" >> ${MESSAGE_PAYLOAD_FILE}
+  #    - name: "Send Message"
+  #      uses: ./.github/actions/slack-file
+  #      with:
+  #        payload-file: ${{ env.MESSAGE_PAYLOAD_FILE }}
+  #        webhook: ${{ secrets.WEBHOOK_COVERAGE }}
+  #    # Disabling for now as this is not critical for job success.   TODO: fix it up.
+  #    #- name: publish to codecov.io
+  #    #  run: bash <(curl -s https://codecov.io/bash) -f $CODECOV_OUTPUT/lcovhtml/lcov.info -F unittest;
+  #    #- uses: ./.github/actions/build-teardown
+
+  transaction-replay:
     runs-on: ubuntu-latest-xl
     container:
       image: libra/build_environment:github-1
     env:
-      CODECOV_OUTPUT: codecov
       MESSAGE_PAYLOAD_FILE: /tmp/message
+      ACCOUNT: ${{ secrets.TRANSACTION_REPLAY_ACCOUNT }}
+      CONTAINER: backup
+      SUB_DIR: e2
+      SAS: ""
+      CONFIG: .github/workflows/transaction-replay.toml
     steps:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/build-setup
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.DIEM_S3_AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.DIEM_S3_AWS_SECRET_ACCESS_KEY }}
-          aws-region: ${{ secrets.DIEM_S3_AWS_REGION }}
-      - name: produce coverage
-        run: cargo xtest --html-cov-dir=$CODECOV_OUTPUT/grcovhtml/ --html-lcov-dir=$CODECOV_OUTPUT/lcovhtml/ --no-fail-fast -j 16 || true
-      - name: Push Coverage Reports to S3
+      - name: install azcopy
```

Comment:
FYI, this can be pre-installed in the CI container in a future follow-up PR
https://github.com/diem/diem/blob/master/docker/ci/github/Dockerfile

Reply:
dev-setup.sh is were the additions go usually.

Reply:
Yeah we can look into doing this in the future, it just takes <1second to do it now so i don't think its a huge deal atm